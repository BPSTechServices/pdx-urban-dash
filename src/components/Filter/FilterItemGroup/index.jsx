import React from "react";
import PropTypes from "prop-types";
import { Container } from "reactstrap";

export default class FilterItemGroup extends React.Component {
  static propTypes = {
    title: PropTypes.string.isRequired,
    hidden: PropTypes.bool
  };

  constructor(props) {
    super(props);

    this.title = "";
    this.hidden = false;
  }

  render() {
    if (this.props.hidden) return null;

    return (
      <Container style={{ marginTop: "1rem" }}>{this.props.children}</Container>
    );
  }
}
