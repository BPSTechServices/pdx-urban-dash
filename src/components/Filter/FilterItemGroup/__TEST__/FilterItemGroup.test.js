import React from 'react';
import { shallow } from 'enzyme';
import FilterItemGroup from '../index';

const testFunc = () => { console.log(); };

describe('FilterItemGroup', () => {
  test('snapshot', () => {
    const wrapper = shallow(<FilterItemGroup title={'test'}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
