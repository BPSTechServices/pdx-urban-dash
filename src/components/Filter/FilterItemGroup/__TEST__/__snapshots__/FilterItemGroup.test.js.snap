// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterItemGroup snapshot 1`] = `ShallowWrapper {}`;
